coalib/results/TextPosition.py

```diff
@@ -7,20 +7,35 @@
 class TextPosition:
 
     @enforce_signature
-    def __init__(self, line: (int, None)=None, column: (int, None)=None):
+    def __init__(self,
+                 line: (int, None)=None,
+                 column: (int, None)=None,
+                 text: (tuple, list, None)=None,
```

Comment:
Before we had line and column and only that info was given. If you want an object with a traditional text position this is not the same information and would have to be an own class. If you want them to be convertible into each other you can give them conversion functions. But please *never* write a quantum state class!